hometask2/hometask2_1.js

```diff
@@ -0,0 +1,145 @@
+const express = require("express");
```

Comment:
Consider using ES6 Imports/Exports, since you're using a babel-node package.
The following article is a bit outdated however it contains a good explanation of the topic:
https://medium.com/@JedaiSaboteur/import-export-babel-and-node-a2e332d15673